.github/workflows/approved-for-ci-run.yml

```diff
@@ -88,15 +99,50 @@ jobs:
             Feel free to review/comment/discuss the original PR #${PR_NUMBER}.
           EOF
 
-          ALREADY_CREATED="$(gh pr --repo ${GITHUB_REPOSITORY} list --head ${BRANCH} --base main --json number --jq '.[].number')"
+          ALREADY_CREATED=${{ steps.get_pr.outputs.ALREADY_CREATED }}
```

Comment:
It's better to move such expressions to `env:`

Reply:
done